src/checkpoint/checkpoint_stats.c

```diff
@@ -23,13 +23,32 @@ __wt_checkpoint_reset_handle_stats(WT_SESSION_IMPL *session, WT_CKPT_CONNECTION
 }
 
 /*
- * __wt_checkpoint_set_handle_stats --
- *     Set handle-related stats.
+ * __wt_checkpoint_reset_time_stats --
+ *     Reset time-related stats.
  */
 void
-__wt_checkpoint_set_handle_stats(
-  WT_SESSION_IMPL *session, WT_CKPT_CONNECTION *ckpt, uint64_t gathering_handles_time_us)
+__wt_checkpoint_reset_time_stats(WT_SESSION_IMPL *session, WT_CKPT_CONNECTION *ckpt)
 {
+    WT_UNUSED(session);
```

Comment:
I think this function doesn't need the `ckpt` arg. Just the session will get you there and that is a cleaner interface. It also gets rid of the `WT_UNUSED` line.

Reply:
Nice, updated in Updated in d4dce6e7d0dd719e8eb5c24fe5b9f138555f7a82